events/triggers/backup

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+BACKUP_ID="1234567890"
+BACKUP_KEY="abcdefghi"
```

Comment:
@nolim1t  I think the idea here is that `BACKUP_ID` and `BACKUP_KEY` are just placeholders. They will be deterministically derived from the LND seed.

I think encrypting backups are important because:
- We can encrypt with a 256 bit key which is likely way stronger than the user's password (this provides significantly better protection against an offline brute force attack)
- This allows us to bundle other arbitrary data into the backup (settings/app data) without leaking it, not just pre-encrypted data like the LND wallet

Reply:
The backup file is useless if the user does not have the correct seed.

Reply:
What if the backup file contains personal information like transaction labels? That could potentially doxx UTXOs if it was leaked.

Reply:
It's already encrypted against the seed and all utxos are on the blockchain on everyone's nodes anyway.

Reply:
I mean extra data, not including the LND backup. So if we included transaction labels, CoinJoin metadata, etc. This is personal data that we would want to include in backups, but that is not encrypted on the local device.

Reply:
Yeah, since we're already bundling `db/user.json` with the backup we can't have it unencrypted as it contains the user's name.